events/office-hours.md

```diff
@@ -62,41 +99,86 @@ This information is for the host/panel to prep the show.
-- [ ] After show: Check and publish the video on youtube, ensure it's in the Office Hours playlist.
+- [ ] -30m before show: Prep a hack.md document with notes and questions from
+      the channel over the course of the last month, ensure panelists have the URL.
+- [ ] 0: Go live, do the show.
+- [ ] After show: Collate links from the show as markdown and post it in the
+      show thread that you posted in the forums.
+- [ ] After show: Check and publish the video on YouTube, ensure it's in the
+      Office Hours playlist.
   
 ### Intro Script
 
-Welcome everyone to today’s Kubernetes Office Hours, where we answer your user questions live on the air with our esteemed panel of experts. You can find us in #office-hours on slack, and check the topic for the URL for the information. 
+Welcome everyone to today’s Kubernetes Office Hours, where we answer your user
+questions live on the air with our esteemed panel of experts. You can find us in
+[#office-hours] on slack, and check the topic for the URL for the information.
 
-- Before we begin let’s start by introducing ourselves: (Give each panelist about a minute)
+- Before we begin let’s start by introducing ourselves: (Give each panelist about
+  a minute)
 - Before we start here are the ground rules:
-  - This is a Kubernetes event so the Code of Conduct is in effect, please be excellent to each other. 
-  - This is a judgement-free zone, everyone had to start from somewhere so please help out your buddy by having a supportive environment in the channel.
-  - While we will do our best to answer your questions the panel doesn’t have access to your cluster, so live debugging is off topic, but we will do our best to get you moving down the next step.
-  - Panelists, you’re encouraged to expand on answers with your experiences and pro-tips.
-  - Audience, you can help by pasting in URLs to official docs, blogs, or anything that might be relevant to the topic at hand. 
-  - Post your questions on discuss.kubernetes.io
-  - You can also help us out by tweeting, spreading the word, and paying it forward. 
-  - This panel is made entirely of volunteers, if you want to rotate in please let us know, we love to have new people rotate in and help out. 
+  - This is a Kubernetes event so the Code of Conduct is in effect, please be
+    excellent to each other.
+  - This is a judgement-free zone, everyone had to start from somewhere so please
+    help out your buddy by having a supportive environment in the channel.
+  - While we will do our best to answer your questions the panel doesn’t have
+    access to your cluster, so live debugging is off topic, but we will do our
+    best to get you moving down the next step.
+  - Panelists, you’re encouraged to expand on answers with your experiences and
+    pro-tips.
+  - Audience, you can help by pasting in URLs to official docs, blogs, or
+    anything that might be relevant to the topic at hand.
+  - Post your questions on [discuss.kubernetes.io]
```

Comment:
No period after the text, unlike all other lines.

Reply:
done 👍